src/selectors/selectors.js

```diff
@@ -41,7 +42,8 @@ export const useRootSelector = <T>(selector: (state: RootReducerState) => T): T
 // Most commonly used selectors
 export const useFiatCurrency = () => useRootSelector(fiatCurrencySelector);
 export const useRates = () => useRootSelector(ratesSelector);
-export const useSupportedAssets = () => useRootSelector(supportedAssetsSelector);
+export const useSupportedAssets = () => useRootSelector(supportedAssetsPerChainSelector);
```

Comment:
Consider matching selector name
```suggestion
export const useSupportedAssetsPerChain = () => useRootSelector(supportedAssetsPerChainSelector);
```